.env

```
@@ -13,3 +13,4 @@ REACT_APP_USE_DOPPLER_LEGACY_SIGNUP=false
 REACT_APP_USE_DOPPLER_LEGACY_FORGOTPASSWORD=false
 NODE_ICU_DATA=node_modules/full-icu
 REACT_APP_ROUTER=browser
+REACT_APP_DOPPLER_UI_LIBRARY_VERSION = 3.51.5
```

What do you think about including the `v` in the version value?

```suggestion
REACT_APP_DOPPLER_UI_LIBRARY_VERSION = v3.51.5
```

---

Another option could be this:

```suggestion
REACT_APP_DOPPLER_UI_BASE_URL = https://cdn.fromdoppler.com/doppler-ui-library/v3.51.5
```